src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -373,25 +373,25 @@ HttpRequestMessage FinalRequestFunc()
                 }
                 catch (Exception ex)
                 {
-                    throw new DaprException(string.Format("ServerErrorNoMeaningFulResponse", response.StatusCode), ex);
+                    throw new DaprApiException(string.Format("ServerErrorNoMeaningFulResponse", response.StatusCode), ex);
```

Comment:
Actors kinda has its own style of errors based on the HTTP interface. Some of the code changed, the functionality should be the same.